client-v2/src/components/FrontsEdit/ArticleFragmentForm.tsx

```diff
@@ -495,6 +458,35 @@ class FormComponent extends React.Component<Props, FormComponentState> {
       </FormContainer>
     );
   }
+
+  private changeImageField = (fieldToSet: string) => {
+    this.allImageFields.forEach(field => {
+      if (field === fieldToSet) {
+        this.props.change(field, true);
+      } else {
+        this.props.change(field, false);
+      }
+    });
+  };
+
+  private getImageFieldName = () => {
+    if (this.props.imageCutoutReplace) {
+      return 'cutoutImage';
+    }
+    return 'primaryImage';
+  };
+
+  private handleImageChange: EventWithDataHandler<React.ChangeEvent<any>> = (
+    e,
+    ...args: [any?, any?, string?]
```

Comment:
do we need this 2nd param? doesn't look like it gets used?

Reply:
strangely, we do (to conform to the `EventWithDataHandler` type), but it's not clear from the code why it's present. I'll see if I can refactor to remove, and if not, I'll add a comment to explain.